### PR TITLE
fix: take address from cli for tictactoe frontend

### PR DESCRIPTION
**Pull request by supercontracts**
**Description**
The tictactoe frontend uses a hardcoded address which may fail on changes. so adding a cli flag `--address=`. 
How to run it: 
```
npm run dev --address=0x123...
# or
npm run build --address=0x123...
```

#274

### CODE REVIEW

File: examples/tictactoe/src/constants/tictactoe.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const address = '0x81D8CB015e36922F7ecC5CEAdd018dc7327e6FC7'
+export const address = import.meta.env.TICTACTOE_ADDRESS || '0x81D8CB015e36922F7ecC5CEAdd018dc7327e6FC7'
```
Comment (hamdiallam) [inline examples/tictactoe/src/constants/tictactoe.ts:1]: If we're going to be supplying the address. Lets instead not have a default provided and simply resort to the cli flag

Is it possible for the process to fail if this flag isn't supplied?

**Review by hamdiallam — CHANGES_REQUESTED** [review]
Thanks for the contribution! This direction makes sense

File: examples/tictactoe/src/constants/tictactoe.ts (hunk 1 above)
Comment (tabshaikh) [reply examples/tictactoe/src/constants/tictactoe.ts]: yeah its possible make a change

**Comment by hamdiallam** [comment]
ty!! will locally test this and merge in after getting off my flight :)

**Review by hamdiallam — CHANGES_REQUESTED** [review]
Can you remove the `yarn.lock` file

**Comment by hamdiallam** [comment]
Looks good! Sorry one last request to also remove the `package-lock.json` file as well since we use pnpm for dependency management and not npm directly

otherwise good to go

**Comment by hamdiallam** [comment]
@tabshaikh npm_config style reading doesn't work for pnpm so superseding this functionality with env vars in #288

**Pull request merged**